ansible/inventories/staging/hosts

```diff
@@ -53,10 +53,12 @@ inventory-web-v1
 inventory-web-v2
 
 [inventory-web-v1]
-inventory[1:2]d.dev-ocsit.bsp.gsa.gov
+inventory1d.dev-ocsit.bsp.gsa.gov usage=with_cron
```

Comment:
Let's use specifically named variables here, like `crons_enabled` like we do for dashboard and wordpress.

Reply:
... Also, the pattern we've been using other places is to not specify variables in the inventory, but instead as host_vars https://github.com/GSA/datagov-deploy/blob/develop/ansible/inventories/production/host_vars/dashboardweb1p.prod-ocsit.bsp.gsa.gov/vars.yml